blobstore/blobstore.go

```diff
@@ -197,3 +246,49 @@ $$ LANGUAGE plpgsql`,
 		`CREATE TRIGGER delete_file BEFORE UPDATE OF deleted_at ON files FOR EACH ROW EXECUTE PROCEDURE delete_file()`,
 	)
 }
+
+type command struct {
+	usage     string
+	f         interface{}
```

Comment:
Make this `func (args *docopt.Args) error` then there is no need for type assertions.